mlir/test/Dialect/MemRef/emulate-narrow-type.mlir

```diff
@@ -203,18 +203,6 @@ func.func @memref_subview_dynamic_offset_i4(%idx : index) -> i4 {
 
 // -----
 
-
-func.func @negative_memref_subview_non_contiguous(%idx : index) -> i4 {
```

Comment:
Could you say why you are removing these tests?

Reply:
I just move them to emulate-narrow-type-unsupported.mlir

Reply:
For the `copy_distinct_layouts`, `memref.alloc() : !colMajor` is illegal, so the error check is changed.

Reply:
Sorry could you please leave the tests where they are. I am not sure it makes sense to split it this way, and it will just be easier to compare the changes (doesnt seem like there is any, but just want to be sure).

Reply:
Okay.